.travis.yml

```diff
@@ -1,19 +1,19 @@
 language: scala
 jdk: openjdk8
-script: sbt -Dfile.encoding=UTF8 -J-XX:ReservedCodeCacheSize=256M "^^${SBT_VERSION}" scripted
+script: sbt -Dfile.encoding=UTF8 -J-XX:ReservedCodeCacheSize=256M "^^${SBT_VERSION}" "scripted ${TEST_GROUPS}"
```

Comment:
This is a nice trick! I've also been looking for a way to run different subsets of scripted tests for different sbt versions, but I couldn't work out how to do it.